fuzzing/private/runtime/clang_runtime_lib.bzl

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Macro that extracts clang runtime libraries from the current cc_toolchain."""
+
+def clang_runtime_lib(*, name, basenames, **kwargs):
+    """Provide the first available clang runtime library with any of the given basenames."""
+    native.genrule(
+        name = name,
+        outs = basenames[:1],
```

Comment:
This part about the output seems less intuitive - perhaps document it in the docstring? 

In particular, it was not clear to me if the output of this rule is a file name or the actual file (library).

Reply:
I now explain this in the docs.